molecularnodes/io/parse/molecule.py

```diff
@@ -439,11 +441,20 @@ def att_vdw_radii():
         vdw_radii = np.array(list(map(
             # divide by 100 to convert from picometres to angstroms which is what all of coordinates are in
             lambda x: data.elements.get(
-                x, {'vdw_radii': 100}).get('vdw_radii', 100) / 100,
+                x, {'vdw_radii': 100}).get('vdw_radii') / 100,
```

Comment:
In checking the tests, `8U8W` contains Iodine, in the file it is written as `IOD` (annoying). Somehow that is causing the `vdw_radii` attribute to fail. This is returning `None` and thus the `None / 100` is failing.`

In theory these backups should be working - but something is failing. If we add back in the fallback `100` it works again. Makes no sense to me - but maybe keeping that fallback there are second time is a good idea (unless you can figure out what is going on).

Reply:
Ah, this error occurs because elements past palladium don't have a `vdw_radii` value explicitly defined in the respective subdictionary in the `elements` dictionary. This chunk of code would fail for any atoms beyond atomic number 47; good that the tests cover one such example! 

The code really should be: 
`vdw_radii = np.array(list(map(lambda x: data.elements.get(x,{}).get('vdw_radii', 100) / 100,` 
where, if the first `.get` call fails, an empty dictionary is provided to the second. If the first `.get` call succeeds but the resulting subdictionary doesn't include a `vdw_radii` keyword, then the default value of 100 is given. Otherwise, the code grabs the defined `vdw_radii` value from the element's subdictionary. 

The similar implementation for `att_mass` doesn't error out for any instances because all subdictionaries in the `elements` dict contain a `standard_mass` key. But, I am going to go ahead and make the two methods similar in format since making assumptions about the underlying data structure could potentially get us into trouble in the future.

Reply:
I can mine vdw radii from https://pubs.acs.org/doi/10.1021/jp8111556  and https://pubs.rsc.org/en/content/articlelanding/2013/dt/c3dt50599e if we need to more completely fill the `elements` subdictionaries. I think there are diminishing returns when worrying about those rarer elements since the likelihood that someone is using MN to visualize a (organo)metallic complex with silver or gold atoms is very small. I could be wrong about that assumption though.

Reply:
If you'd like to include as many details for vdw radii as possible then do so - otherwise I can spend some time filling out the dictionary at a later date. Like you said, super unlikely that it would happen, but I'd like to eventually cover it to head off weird bugs in the future that I can't track down ahead of time.

Reply:
Yeah, filling in VDW radius values for these elements is very low priority for me. [MDAnalysis](https://docs.mdanalysis.org/2.7.0/documentation_pages/topology/tables.html#MDAnalysis.topology.tables.vdwradii) doesn't even list a complete table of vdw radii. 

The code change above should avoid any fatal errors that might arise due to missing info. I can create an Issue focused on filling in the remaining elements' values. I'll include the link to the papers that I've pulled up. It certainly would be a 'good first issue' for someone to complete. 

Just out of curiosity, when do `vdw_radii` attribute values get used? For the spheres/balls-and-stick representations? For surface representations?

Reply:
If you could open an issue with the links that would be great!

You are right it's used fro spheres / ball & stick / surface style, and also for finding bonds with nodes. Otherwise it isn't used internally at the moment.